site/docs/plugins/commands.md

```diff
@@ -492,15 +492,15 @@ bot
   .filter(commandNotFound(myCommands))
   // If so, that means it wasn't handled by any of our commands.
   .use(async (ctx) => {
-    // We found a potential match
     if (ctx.commandSuggestion) {
+      // We found a potential match
       await ctx.reply(
         `Hmm... I don't know that command. Did you mean ${ctx.commandSuggestion}?`,
       );
+    } else {
+      // Nothing seems to come close to what the user typed
+      await ctx.reply("Oops... I don't know that command :/");
     }
```

Comment:
I'm a strong believer in early return over `else`. Please apply this to the other snippet too :)

```suggestion
    if (ctx.commandSuggestion) {
      // We found a potential match
      return ctx.reply(
        `Hmm... I don't know that command. Did you mean ${ctx.commandSuggestion}?`,
      );
    }
    
    // Nothing seems to come close to what the user typed
    await ctx.reply("Oops... I don't know that command :/");
```